# --- T2-COPYRIGHT-BEGIN ---
# t2/package/*/clisp/hotfix-makemake.patch
# Copyright (C) 2025 The T2 SDE Project
# SPDX-License-Identifier: GPL-2.0 or patched project license
# --- T2-COPYRIGHT-END ---

--- clisp-2.49/src/makemake.in.vanilla	2025-04-15 15:35:54.079085243 +0200
+++ clisp-2.49/src/makemake.in	2025-04-15 15:37:13.878725978 +0200
@@ -3317,7 +3317,7 @@
 # is bash, config.cache may be unparsable with sh on Solaris
 echotab "      \$(SHELL) \$\$m/configure --with-clisp=\"${HEREP}/clisp -K boot ${someflags}\" --cache-file=\$\${cache} --srcdir=\$\$m \$(MODULE_CONFIGURE_FLAGS);\\"
 echotab "    else \\"
-echotab "      \$(SHELL) \$\$m/configure --srcdir=\$\$m \$(MODULE_CONFIGURE_FLAGS); \\"
+echotab "      \$(SHELL) \$\$m/configure --with-clisp=\"${HEREP}/clisp -K boot ${someflags}\" --srcdir=\$\$m \$(MODULE_CONFIGURE_FLAGS); \\"
 echotab "    fi ) ;\\"
 echotab "fi; \\"
 # srcdir must be correct in Makefile because clisp-link calls link.sh
